bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -108,10 +108,6 @@ class VideoProvider extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {
-      socketOpen: false,
```

Comment:
Do not remove this state. It may be flagged as "unused", but it's not useless.
The state is there to force a re-run of `componentDidUpdate` when the signaling socket
opens or closes.
It's not pretty, but it's needed due to how this stuff is currently laid out.

I recommend checking whether the other states you're removing are truly useless as well (I can't say).

Reply:
Oooh, ok! I'll restore it and add a comment